runtime/src/main/kotlin/kotlin/coroutines/CancellationException.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package kotlin.coroutines.cancellation
+
+/**
+ * Thrown by cancellable suspending functions if the coroutine is cancelled while it is suspending.
+ * It indicates _normal_ cancellation of a coroutine.
+ */
+public open class CancellationException(
```

Comment:
For consistency please provide the following set of constructors:
https://github.com/JetBrains/kotlin/blob/b06780988df1e9cf207d3b663017138aff0e665e/libraries/stdlib/common/src/kotlin/ExceptionsH.kt#L118-L121